<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>974</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(123, 233, 255, 255), stop:1 rgba(255,255,255 255))
</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="maximumSize">
       <size>
        <width>1360</width>
        <height>680</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="1" column="0" rowspan="4">
         <widget class="QTextBrowser" name="display_terminal">
          <property name="styleSheet">
           <string notr="true">background-color : black; color : white</string>
          </property>
          <property name="html">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1" colspan="2">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QPushButton" name="DAC">
            <property name="text">
             <string>Perform DAC</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="ADC">
            <property name="text">
             <string>Perform ADC</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="4" column="1" colspan="2">
         <layout class="QGridLayout" name="gridLayout">
          <item row="4" column="0">
           <widget class="QRadioButton" name="fw">
            <property name="text">
             <string>Forward wave</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QRadioButton" name="sw">
            <property name="text">
             <string>Square wave</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QRadioButton" name="zz">
            <property name="text">
             <string>ZIG-ZAG wave</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QRadioButton" name="stw">
            <property name="text">
             <string>Stepping Wave</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_6">
          <property name="font">
           <font>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="text">
           <string>8255: DAC-ADC</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1" colspan="2">
         <widget class="QPushButton" name="setvoltage">
          <property name="text">
           <string>Set Voltage</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLineEdit" name="terminal">
          <property name="styleSheet">
           <string notr="true">background-color : black; color : white</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_9">
          <property name="styleSheet">
           <string notr="true">background-color: rgba(94, 255, 148, 0);</string>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Developed By : Niranjan Rathod (SE-4)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" rowspan="2" colspan="2">
         <widget class="QPushButton" name="about">
          <property name="text">
           <string>About App</string>
          </property>
         </widget>
        </item>
        <item row="4" column="3">
         <widget class="QLabel" name="blockdigs">
          <property name="text">
           <string>TextLabel</string>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2">
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>20</y>
          <width>400</width>
          <height>400</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgba(255, 255, 255, 0);
</string>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_3">
       <widget class="QLCDNumber" name="lcdNumber">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>120</y>
          <width>231</width>
          <height>81</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color:red;background-color:black
</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="smallDecimalPoint">
         <bool>true</bool>
        </property>
        <property name="digitCount">
         <number>3</number>
        </property>
        <property name="segmentStyle">
         <enum>QLCDNumber::Flat</enum>
        </property>
       </widget>
       <widget class="QDial" name="dial">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>220</y>
          <width>131</width>
          <height>101</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="ret">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>20</y>
          <width>99</width>
          <height>27</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;-- Return</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>60</y>
          <width>81</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>UnDinaru</family>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>VOLTAGE</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_4">
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>-30</y>
          <width>561</width>
          <height>551</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#4210d8;&quot;&gt;Copy All Image Files From Project Directory to build directory.&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; color:#000000;&quot;&gt;Best viewd in 1080p&lt;/span&gt;&lt;span style=&quot; color:#dace1d;&quot;&gt;.&lt;/span&gt;&lt;/p&gt;&lt;p&gt;This application is developed to simulate the &lt;span style=&quot; color:#ff0000;&quot;&gt;Environment of&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;MIT KIT Assignment Practicals &lt;/span&gt;&lt;span style=&quot; color:#000000;&quot;&gt;for those who haven't practiced them well in Lab.&lt;/span&gt;&lt;/p&gt;&lt;p&gt;These Applications are rapidly developed within duration of&lt;span style=&quot; color:#00aa00;&quot;&gt; two&lt;/span&gt; days.&lt;/p&gt;&lt;p&gt;Therefore this application may not meet &lt;span style=&quot; color:#00aa00;&quot;&gt;expection&lt;/span&gt; of many users.&lt;/p&gt;&lt;p&gt;Still having something to practise is always better than having nothing.&lt;/p&gt;&lt;p&gt;Note that &lt;span style=&quot; color:#00aa00;&quot;&gt;Only 8251&lt;/span&gt; have 'Simulated Compiler' support. &lt;/p&gt;&lt;p&gt;So be Careful and type ONLY valid codes. If you are facing any problem &lt;/p&gt;&lt;p&gt;with this application &lt;/p&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;1]&lt;/span&gt; Please Use &lt;span style=&quot; color:#ff0000;&quot;&gt;sample code&lt;/span&gt; given in Project directoy.&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;2] &lt;/span&gt;Or refer to &lt;span style=&quot; color:#ff0000;&quot;&gt;video &lt;/span&gt;given in same directory.&lt;/p&gt;&lt;p&gt;and if you like to develop things you are always &lt;span style=&quot; color:#00aa00;&quot;&gt;welcome to contribute&lt;/span&gt;&lt;/p&gt;&lt;p&gt;to repository.&lt;/p&gt;&lt;p&gt;-Niranjan Rathod &lt;/p&gt;&lt;p&gt;For Details and bugs report and reviews :P: Whatsapp at 7276757446&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
